packages/xsnap/src/xsrepl.js

```diff
@@ -97,4 +97,4 @@ async function main() {
   return vat.close();
 }
 
-main();
+await main();
```

Comment:
Oh, right, can we use top-level `await` now? Nice, I haven't internalized that yet.

I'd go one step further and do `await main().catch(err => console.log(err))`, because in my experience, simple errors or typos in functions called like this `main()` get lost, and the explicit `catch` is the only way to discover the problem.